css/styles.css

```diff
@@ -0,0 +1,778 @@
+}
+
+.featured-card {
+  display: flex;
+  margin: 1rem;
+  padding: 1rem;
+  transition: transform 300ms, outline 300ms;
+}
+
+.featured-card:hover {
+  transform: scale(1.03);
+  outline: 1px solid #f30;
+}
+
+.featured-content {
+  margin: 0 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.person-name {
+  font-size: 1.4em;
+  font-weight: 900;
+}
+
+.person-ocupation {
+  color: #f30;
+  font-weight: 600;
+  font-style: italic;
+}
+
+.featured-divisor {
+  height: 1px;
+  background: #f30;
+  width: 15%;
+  margin: 1rem 0;
+}
+
+.featured-btn-container {
+  display: flex;
+  justify-content: center;
+  width: 50%;
+}
+
+.more-btn {
+  margin: 1rem auto;
+  width: 100%;
+  padding: 1rem;
+  background: #fff;
+  border: 1px solid #f30;
+  font-weight: 600;
+  transition: background 300ms, color 300ms;
+}
+
+.more-btn:hover {
+  background: #f30;
+  color: #fff;
+  cursor: pointer;
+}
+
+.fa-chevron-down {
+  color: #000;
+  margin: 0 0.5rem;
+}
+
+/** LOGO SECTION */
+.logo-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.logo-section-title {
+  font-size: 2.2em;
+  margin: 3rem auto 1rem auto;
+  width: 60%;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+  font-weight: 600;
+  color: #444;
+}
+
+.line {
+  width: 4rem;
+  height: 1px;
+  margin: 0 auto 1rem auto;
+  background: #f30;
+}
+
+.logo-description {
+  width: 70%;
+  text-align: center;
+  margin: 0 auto;
+  color: #444;
+}
+
+.logo-icon {
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  margin: 3rem auto;
+  width: 65%;
+}
+
+/** PAST SECTION  */
+.past-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.past-title {
+  color: #555;
+  font-size: 2.2em;
+  margin: 3rem auto 1rem auto;
+  font-weight: 600;
+  text-align: center;
+}
+
+.past-description {
+  width: 70%;
+  text-align: center;
+  color: #444;
+  margin: 0 0 2rem 0;
+}
+
+.past-container {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.past-container > img:hover {
+  transform: scale(1.06);
+}
+
+/** PARTNERS SECTION  */
+.partners-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: #444;
+}
+
+.partners-title {
+  color: #aaa;
+  font-weight: 900;
+  font-size: 2.5em;
+  margin: 1rem 0 0 0;
+  font-family: 'cocogoose', sans-serif;
+}
+
+.partners-container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.partners-container > img:hover {
+  background: #eee;
+}
+
+/** FOOTER */
+footer {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  padding: 1rem;
+}
+
+.footer-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.footer-container > p {
+  margin: 1rem 0;
+  text-align: center;
+}
+
+/** FOOTER ABOUT */
+.footer-about {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
- [ ] As required on the project the About Page it has a dark background and the Home Page has a white background, please kindly make the changes 😁

Your Home page
<img width="1022" alt="Screen Shot 2022-09-15 at 01 58 21" src="https://user-images.githubusercontent.com/74501523/190283329-533632c1-dc97-4897-beb4-227acb61af3f.png">

Your About page

<img width="1022" alt="Screen Shot 2022-09-15 at 01 58 06" src="https://user-images.githubusercontent.com/74501523/190283365-d843a335-989e-49de-a714-c08a218c5b13.png">